ts/theme/components/Text.ts

```diff
@@ -9,6 +9,11 @@ declare module "native-base" {
     interface Text extends TextProperties {
       link?: boolean;
       bold?: boolean;
+      dateFormat?: boolean;
```

Comment:
`dateFormat` looks like this is the format of the date, I guess this is more like `formatDate`?